security/s2a/internal/conn/expander.go

```diff
@@ -0,0 +1,10 @@
+package conn
+
+import "hash"
+
+// HKDFExpander is the interface for an HMAC-based Extract-and-Expand Key
+// Derivation Function (HKDF) expander. This is specified in Section 7 of
```

Comment:
Can you change `Section 7` to `Section 7.2`? And make the link to that section instead? I.e.

https://tools.ietf.org/html/rfc8446#section-7.2

Reply:
Also, can you change the start of the second sentence to `It's use in TLS 1.3 is specified in...`?

Reply:
Done